dipper/sources/ZFINSlim.py

```diff
@@ -19,13 +19,13 @@ class ZFINSlim(Source):
     """
     files = {
         'g2p_clean': {
-            'file': 'phenoGeneCleanData_fish.txt.txt',
+            'file': 'phenoGeneCleanData_fish.txt',
             'url': 'https://zfin.org/downloads/phenoGeneCleanData_fish.txt',
             # https://zfin.org/downloads#  header Documentation is burried in UI crap
         },
         'zpmap': {
-            'file': 'zp-mapping.txt',
-            'url': 'http://compbio.charite.de/hudson/job/zp-owl-new/lastSuccessfulBuild/artifact/zp.annot_sourceinfo'
+            'file': 'zp-mapping-2019.txt',
```

Comment:
Why is the year in here? Will this not be superseded very quickly?

Reply:
Basically to stop from using the old files while I was testing, and also to indicate when the mapping was made. Easy enough to change

Reply:
sure doesnt matter.